pcweb/pages/docs/component.py

```diff
@@ -30,10 +39,49 @@ class Prop(Base):
     # The description of the prop.
     description: str
 
+    # The default value of the prop.
+    default_value: str
 
-from reflex.components.el.elements.base import BaseHTML
 
+from reflex.components.el.elements.base import BaseHTML
+import re
 
+def get_default_value(lines: list[str], start_index: int) -> str:
+    """Process lines of code to get the value of a prop, handling multi-line values.
+    Args:
+    lines: The lines of code to process.
```

Comment:
nit: arguments / return should be indented one level in. Also include new lines between description / args / return